composer.json

```diff
@@ -6,7 +6,7 @@
   "license": "GPL-2.0-or-later",
   "minimum-stability": "dev",
   "require": {
-    "drupal/webform": "^6.0",
+    "drupal/webform": "^6.3@beta",
```

Comment:
Would we be able to stick to `^6.2` here? Just that this involves installing a beta that has quite a bit of rewrites. To install localgov_forms I think you need a minimum stability of dev anyway so it should allow webform 6.3 beta to install.

Reply:
I assume you mean `^6.2 || ^6.3@beta`?

The main difference between the two, at least at the beginning seems to have been 11 support. I've not checked how far they've diverged since.

Reply:
Agree that the `^6.2` requirement should install 6.2 and 6.3 beta, but no harm in explicitly stating this, so have updated the requirement with Ekes' suggestion.